package.json

```diff
@@ -1,13 +1,12 @@
 {
+  "name": "dadbot",
   "dependencies": {
     "@discordjs/opus": "^0.5.3",
     "@discordjs/voice": "^0.7.5",
+    "csv-parse": "^5.0.4",
     "discord-tts": "^1.2.1",
     "discord.js": "^13.5.1",
-    "dotenv": "^12.0.0",
-    "ffmpeg": "^0.0.4",
```

Comment:
Oh right - I didn't mean to remove ffmpeg, ffmpeg-static, libsodium-wrappers - if those are necessary, LMK so I can put them back in.

Reply:
Yeah so I'm using dotenv to pull the discord API key into an environment variable from the .env file. I don't know if that's the best way to handle that, but that's what it's for. The ffmpeg and libsodium-wrappers package are used for the TTS in voice channels. I don't remember if ffmpeg-static is actually necessary though, I may have just forgot to remove it. But the TTS message has to be converted to an audio file before it can be played by the bot.

Reply:
Eh fuck it, let me amend this commit and put those back in

I ended up putting in way more dependencies than I actually needed trying to write the scraper on the server side, but took out ones that were there before (whoops) when cleaning things up.